lib/rubycritic/commands/default.rb

```diff
@@ -20,6 +22,15 @@ def critique
         RevisionComparator.new(@paths).set_statuses(analysed_modules)
       end
 
+      def open(report_path = nil)
```

Comment:
ControlArgumentSmell. Lets move this whole logic to the caller since he already knows what to do.